Develop

xschildw opened the pull request:
Repo svc stack version.

xschildw committed:
Repo service versionInfo

xschildw committed:
New version

xschildw committed:
Updated

xschildw committed:
Throw exception if file not found.

xschildw committed:
Added client API + VersionInfo pogo instead of string.

xschildw committed:
Merge branch 'PLFM-1317' into develop

xschildw committed:
Basic setup for synapsePython library

RadicalZephyr committed:
Remove the LogData class.  Use String.format instead

RadicalZephyr committed:
Simplify ActivityLogger's pointcut setup

Move all of the pointcut's used into the ActivityLogger, and
delete the SystemArchitecture class altogether.  Also,
made the execution pointcut wildcard for all Classes in the
controller package.

RadicalZephyr committed:
Changes to logging configuration

All file appenders now use the log4j extras RollingFileAppender.
This is because of potential synchronization issues with using the
DailyRollingFileAppender.

john-hill commented:
This pull request cannot be automatically merged.  Can you resolve the conflict and push again.

xschildw committed:
Fix conflict

RadicalZephyr committed:
Remove annotations logging capability from ActivityLogger

Possibly a simpler form where we just log what the class of
annotation could be useful, but it's not necessary right
now.

RadicalZephyr committed:
Change ActivityLogger output format to URL query string

This will facilitate maintaining human readability and
ease of parsing.

RadicalZephyr committed:
Cleanups to bean definition and logging properties.

xschildw committed:
Fixed conflict

xschildw committed:
Merge branch 'develop' of https://github.com/Sage-Bionetworks/Synapse-Repository-Services into develop

# By jay-hodgson
# Via MattNapsAlot (1) and jay-hodgson (1)
* 'develop' of https://github.com/Sage-Bionetworks/Synapse-Repository-Services:
  SWC-137: Release version of the fix, develop already has this fix. When portal SynapseClientImpl.getUnmetAccessRequirements() pulls UserProfile for the principle ID (when anonymous), this error occurs.  Now return a relatively empty UserProfile with the right principle ID for the anonymous user, instead of throwing an error.

RadicalZephyr committed:
Rename TestClass and move to the controller package

Also added the @Controller annotation to facilitate the coming
autowired test.

RadicalZephyr committed:
Add a test to verify the spring configuration is setup correctly

The test validates that the ActivityLogger is indeed called
before the method invocations targeted by it's pointcut.

RadicalZephyr committed:
Enhance documentation for ActivityLogger

RadicalZephyr committed:
Add more tests for ActivityLogger

john-hill committed:
Merge pull request #36 from ezephyr/develop

Completion of services logging feature

john-hill committed:
Merge pull request #37 from brucehoff/develop

PLFM-1399

xschildw committed:
Fixed conflicts
